src/Mono.Android/Test/Properties/AndroidManifest.xml

```diff
@@ -1,7 +1,7 @@
 ﻿<?xml version="1.0" encoding="utf-8"?>
 <manifest xmlns:android="http://schemas.android.com/apk/res/android" android:versionCode="1" android:versionName="1.0" package="Mono.Android_Tests">
 	<uses-sdk android:minSdkVersion="16" android:targetSdkVersion="28" />
-	<application android:label="Xamarin.Android.RuntimeTests" android:name="android.apptests.App" android:usesCleartextTraffic="true">
+	<application android:label="Xamarin.Android.RuntimeTests" android:name="android.apptests.App" android:usesCleartextTraffic="true" android:debuggable="true">
```

Comment:
[As per above](https://github.com/xamarin/xamarin-android/pull/3753/files#r340628833), this change isn't necessary.  This isn't the app that we need to enable GREF logging on.